plugin/plugin.go

```diff
@@ -42,14 +43,16 @@ var wellKnownTypes = map[string]string{
 }
 
 type OrmableType struct {
-	Name    string
-	Package string
-	Fields  map[string]*Field
+	OriginName string
+	Name       string
+	Package    string
+	Fields     map[string]*Field
 }
 
 type Field struct {
 	Type string
 	*gorm.GormFieldOptions
+	BindTo string
```

Comment:
Could you make it `Parent *OrmableType` so it can be used a bit more universally across the plugin?